erpnext/manufacturing/doctype/work_order/work_order.js

```diff
@@ -646,15 +646,26 @@ erpnext.work_order = {
 
 	show_prompt_for_qty_input: function(frm, purpose) {
 		let max = this.get_max_transferable_qty(frm, purpose);
```

Comment:
get_max_transferable_qty will return FG Qty, you need to change it to RM Qty for Process Manufacturing

Reply:
we are not using 'get_max_transferable_qty' if manufacturing type is process

Reply:
Why ?